snippets/contentSecurityPolicy.cson

```diff
@@ -0,0 +1,18 @@
+# useful for eliminating the errors in console when using ember-cli
+
+'.source.js':
+
+  'Content Security Policy':
```

Comment:
I don't think we should have that. It might be helpful to some people, it's just not something that you write constantly.